robottelo/ui/discoveryrules.py

```diff
@@ -68,10 +67,6 @@ def search(self, name):
         """Searches existing discovery rule from UI. It is necessary to use
         custom search as we don't have both search bar and search button there.
         """
-        if not bz_bug_is_open(1233135):
```

Comment:
That one - http://projects.theforeman.org/issues/9021?
so something like `rm_bug_is_open(9021)`?